cmd/server_test.go

```diff
@@ -1843,6 +1843,8 @@ func (s *TestSuiteCommon) TestGetPartialObjectMisAligned(c *C) {
 	1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,
 	1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,
 	1234567890,1234567890,1234567890,123`
+	// ensuring that the buffer contains atleast 13 bytes of data.
+	buffer.WriteString(line[:13])
```

Comment:
[Here](https://play.golang.org/p/wZxs9e1Auw) is an implementation that provides repeatable in-place randomization for byte-arrays.

The repeatability is useful because it ensures that in every run the same sequence of random values are generated - without having repeatable random numbers, if a test finds a bug, it will not be easily reproducible because in the next run the random numbers would change and the bug may not manifest.